web/cobrands/shropshire/_colours.scss

```diff
@@ -3,37 +3,37 @@
 $container-max-width: 1024px;
 
 /* COLOURS */
-
-$shropshire-blue: #007DB8; // primary brand colour
+$shropshire-dark-blue: #132f65; // CTA buttons in header/nav
 $shropshire-royal-blue: #2266CA; // links
+$shropshire-blue: #2A79AB; // primary brand colour
+$shropshire-blue-l1:#2583c5; // border
 $shropshire-pale-blue: #87CEEB; // footer links
-$shropshire-dark-blue: #005785; // CTA buttons in header/nav
-$shropshire-green: #008A0E; // CTA buttons in body content
+
+$shropshire-green: #00873f; // CTA buttons in body content
 $shropshire-mid-grey: #555555; // headings and paragraphs
 $shropshire-dark-grey: #2C2C2C; // focussed button bg, hovered primary button bg
 $shropshire-yellow: #EEB92A; // focus outlines
 $shropshire-red: #D43531; // destructive actions
 
 $primary: $shropshire-blue;
-$primary_b: #000000;
+$primary_b: #333;
 $primary_text: $shropshire-mid-grey;
 
 $base_bg: #f4f4f4;
 $base_fg: $shropshire-mid-grey;
 
-$nav_background_colour: $shropshire-blue;
+$nav_background_colour: $shropshire-dark-blue;
 $nav_colour: #fff;
 $nav_hover_background_colour: $shropshire-dark-blue;
 
 $header-top-border: false;
 
 // Colour used for front page 'how to report a problem' steps
-$col_big_numbers: $shropshire-blue;
-
+$col_big_numbers: $shropshire-green;
 $col_click_map: $shropshire-blue;
 
-$body-font: 'myriad-pro', 'Helvetica Neue', Arial, Helvetica, 'Segoe UI', sans-serif;
-$heading-font: $body-font;
+$body-font: 'Helvetica Neue', Arial, Helvetica, 'Segoe UI', sans-serif;
+$heading-font: 'myriad-pro', 'Helvetica Neue', Arial, Helvetica, 'Segoe UI', sans-serif;;
```

Comment:
Are you sure these two font lines are right / needed? I can’t see where we’re loading either Myriad Pro or Helvetica Neue as web fonts, so I assume we’re relying on them being installed on the user’s device … but the spelling of `myriad-pro` (rather than `Myriad Pro`) makes me think it’d be pretty unlikely to match _even_ if the user’s device has Myriad Pro available.

Are these the `font-family` lines that Shropshire defined in the template they sent over?

Reply:
Thanks @Zarino. I didn't realise of that. I just made a commit including the Myriad family. I have included it the same way the templates have. However, the typekit links are broken, I don't think they client has noticed it, because in their website they are using Helvetica, but 'myriad' has been defined. So I'll ask they client to provide the links, because is not a major thing I think we can just launch it on the staging website and update the links once they client has provided them.